### PR TITLE
[KOGITO-870] Operator: Spring Boot Persistence not working

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-870

Environment variables has been corrected/added.
Cucumber tests is now enabled.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by ricardozanini** [comment]
@radtriste please add a unit test to verify this change?

File: pkg/controller/kogitoapp/resource/deployment_config.go [hunk 1]
```diff
@@ -192,6 +193,7 @@ func SetInfinispanEnvVars(cli *client.Client, kogitoInfra *v1alpha1.KogitoInfra,
 			vars := envVarInfinispanQuarkus
 			if kogitoApp.Spec.Runtime == v1alpha1.SpringbootRuntimeType {
 				vars = envVarInfinispanSpring
+				framework.SetEnvVar(envVarInfinispanSpringUseAuth, "true", &dc.Spec.Template.Spec.Containers[0])
```
Comment (ricardozanini) [inline pkg/controller/kogitoapp/resource/deployment_config.go]: I guess we can follow the same pattern as the Quarkus map and also add the var [`QUARKUS_INFINISPAN_CLIENT_USE_AUTH`](https://quarkus.io/guides/infinispan-client#quarkus-infinispan-client_quarkus.infinispan-client.use-auth). This way this line would be outside this conditional.

**Comment by radtriste** [comment]
did a mistake in last push ... will update that ...

**Comment by radtriste** [comment]
Now you can review :)

File: pkg/controller/kogitoapp/resource/deployment_config.go (hunk 1 above)
Comment (radtriste) [reply pkg/controller/kogitoapp/resource/deployment_config.go]: added

**Pull request merged**